scripts/install.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+set -euo pipefail
+
+# RAPIDS_CUDA_VERSION is like 12.15.1
+# We want cu12
+RAPIDS_PY_CUDA_SUFFIX=$(echo "cu${RAPIDS_CUDA_VERSION:-12.15.1}" | cut -d '.' -f 1)
+
+DASK_VERSION=main
+
+# Try
+uv pip install --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple \
+  --overrides=requirements/overrides.txt \
```

Comment:
This `--overrides` is new. It will ensure we still get Dask main even when we pin rapids-dask-dependency.